src/App.tsx

```diff
@@ -10,33 +11,28 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabIdState, setSelectedTabIdState] = useState(tabs[0].id);
+  const [selectedTabTitle, setSelectedTabTitle] = useState(tabs[0].title);
+  const [selectedTabContent, setSelectedTabContent] = useState(tabs[0].content);
```

Comment:
Do you really need to have a state for each tab's property?
You can have only state for tab(or tabId), and just get its properties where needed